AutoSteamApp/Automaton/SteamworkAutomaton.cs

```diff
@@ -0,0 +1,348 @@
+﻿using AutoSteamApp.Configuration;
+using AutoSteamApp.Helpers;
+using AutoSteamApp.ProcessMemory;
+using GregsStack.InputSimulatorStandard;
```

Comment:
The "master" branch does not compile as you deleted the nuget packages for GregsStack.InputSimulator. You will need to restore the packages.config file.

Reply:
I will check my git settings. On my local work the package is still there so I am currently going to look into why it is not being pushed.

Reply:
Looking into the gitignore, the "/packages" folder is supposed to be ignored. Do you want me to edit the gitignore so that the packages are pushed?

Reply:
Nope, the packages folder should never be committed or pushed. The restore is done by VS by enabling it in settings or manually by right click project -> Restore NuGet Packages

Reply:
Before all those changes, you deleted the packages.config, thus making VS unable to restore them. That is why it didn't work